packages/xo-web/src/xo-app/new/network/index.js

```diff
@@ -63,10 +63,12 @@ If and only if the PIF:
 For more info see: https://xapi-project.github.io/xapi/design/tunnelling.html
 */
 const canSupportPrivateNetwork = (pool, pif) =>
-  (pif.isBondMaster || pif.physical || pif.vlan !== -1) &&
-  pif.mode !== 'None' &&
-  !pif.isBondSlave &&
-  pif.$host === pool.master
+  pool === undefined
```

Comment:
IMHO, you also can check for `pif === undefined`, to be sure `pif.<sub-prop>` do not generate another error.
```suggestion
  pool === undefined || pif === undefined
```